.github/workflows/MainDistributionPipeline.yml

```
@@ -14,7 +14,7 @@ concurrency:
 jobs:
   duckdb-next-build:
     name: Build extension binaries
-    uses: duckdb/extension-ci-tools/.github/workflows/_extension_distribution.yml@main
+    uses: ./.github/workflows/_extension_distribution.yml@main
```

actionlint (1.7.7)

reusable workflow call "./.github/workflows/_extension_distribution.yml@main" at "uses" is not following the format "owner/repo/path/to/workflow.yml@ref" nor "./path/to/workflow.yml". see https://docs.github.com/en/actions/learn-github-actions/reusing-workflows for more details

(workflow-call)